packages/vue/src/components/molecules/SfBar/SfBar.vue

```diff
@@ -1,38 +1,38 @@
 <template>
   <div class="sf-bar">
     <slot name="back">
-      <div class="sf-bar__icon">
-        <SfIcon
-          v-if="back"
-          icon="chevron_left"
-          size="14px"
-          role="button"
-          aria-label="back"
-          @click="$emit('click:back')"
-        />
-      </div>
+        <button v-focus class="sf-bar__icon">
+          <SfIcon
+            v-if="back"
+            icon="chevron_left"
+            size="14px"
+            aria-label="back"
+            @click="$emit('click:back')"
+          />
+        </button>
     </slot>
     <slot name="title" v-bind="{ title }">
       <div class="sf-bar__title">{{ title }}</div>
     </slot>
     <slot name="close">
-      <div class="sf-bar__icon">
-        <SfIcon
-          v-if="close"
-          icon="cross"
-          size="14px"
-          role="button"
-          aria-label="close"
-          @click="$emit('click:close')"
-        />
-      </div>
+        <button v-focus class="sf-bar__icon">
+          <SfIcon
+            v-if="close"
```

Comment:
`v-if` should be on button, the same as `aria-label` and click method. Please check other icons :)